### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rebeccalee828**

### CODE REVIEW

**Review by rebeccalee828 — COMMENTED** [review]
Hi Dennis, I updated the README with the wording changes I already reviewed.

File: README.md [hunk 1]
```diff
@@ -72,17 +72,20 @@
 | 12      | TELEMETRY_WMI              |
 | 13      | TELEMETRY_MEMORY           |
 | 14      | TELEMETRY_BM               |
+| 17      | TELEMETRY_EVENT_PIPE       |
```
Comment (trend-dennis-tseng) [inline README.md:75]: `TELEMETRY_EVENT_ID` in the proto setting may need update to this:

```
    TELEMETRY_APP              = 15;  // mobile only
    TELEMETRY_SYSTEM_EVENT     = 16;  // mobile only
    TELEMETRY_PIPE             = 17;
    TELEMETRY_MAC_SYS_LOG      = 18;
    TELEMETRY_DDR              = 19;
    TELEMETRY_ASSOCIATION      = 101; // rca association usage
```

File: README.md [hunk 2]
```diff
@@ -92,6 +95,8 @@
 | 107        | TELEMETRY_FILE_CLOSE                           |
 | 108        | TELEMETRY_FILE_MODIFY_TIMESTAMP                |
 | 109        | TELEMETRY_FILE_MODIFY                          |
+| 110        | TELEMETRY_FILE_SET_ATTRIBUTES                  |
+| 111        | TELEMETRY_FILE_ENUMERATE                       |
```
Comment (trend-dennis-tseng) [inline README.md:99]:
```
    TELEMETRY_FILE_SET_EXTENDED_ATTRIBUTE       = 112; //mac os set extended attributes
    TELEMETRY_FILE_DELETE_EXTENDED_ATTRIBUTE    = 113; //mac os delete extended attributes
```

File: README.md [hunk 3]
```diff
@@ -101,6 +106,10 @@
 | 402        | TELEMETRY_REGISTRY_SET                         |
 | 403        | TELEMETRY_REGISTRY_DELETE                      |
 | 404        | TELEMETRY_REGISTRY_RENAME                      |
+| 405        | TELEMETRY_REGISTRY_ENUMERATE                   |
+| 406        | TELEMETRY_REGISTRY_ENUMERATEVALUE              |
+| 407        | TELEMETRY_REGISTRY_QUERYVALUE                  |
+| 408        | TELEMETRY_REGISTRY_SAVE                        |
```
Comment (trend-dennis-tseng) [inline README.md:112]:
```
TELEMETRY_DNS_QUERY = 301;

File: README.md [hunk 4]
```diff
@@ -120,6 +129,8 @@
 | 1003       | TELEMETRY_MEMORY_READ                          |
 | 1101       | TELEMETRY_BM_INVOKE                            |
 | 1102       | TELEMETRY_BM_INVOKE_API                        |
+| 1401       | TELEMETRY_PIPE_CREATE                          |
+| 1402       | TELEMETRY_PIPE_CONNECT                         |
```
Comment (trend-dennis-tseng) [inline README.md:133]:
```
    TELEMETRY_MEMORY_READ              = 1003;

    TELEMETRY_BM_INVOKE     = 1101;
    TELEMETRY_BM_INVOKE_API = 1102;

    TELEMETRY_APP_START                            = 1201; // mobile only
    TELEMETRY_APP_STOP                             = 1202; // mobile only
    TELEMETRY_APP_INSTALL                          = 1203; // mobile only
    TELEMETRY_APP_UNINSTALL                        = 1204; // mobile only
    TELEMETRY_APP_BEHAVIOR                         = 1205;

    TELEMETRY_SYSTEM_EVENT_ENABLE             = 1301;  // mobile only
    TELEMETRY_SYSTEM_EVENT_DISABLE            = 1302;  // mobile only
    TELEMETRY_SYSTEM_CERTIFICATION_INSTALL    = 1303;
    TELEMETRY_SYSTEM_DEVICE_ROOTED            = 1304;

    TELEMETRY_PIPE_CREATE  = 1401;
    TELEMETRY_PIPE_CONNECT = 1402;

    TELEMETRY_MAC_SYS_LOG_COLLECT = 1601;

    // rca association usage
    TELEMETRY_ASSOCIATION_PROCESS_IMAGE_FILE        = 10101;
    TELEMETRY_ASSOCIATION_AUTO_RUN_KEY_FULL_PATH    = 10102;
    TELEMETRY_ASSOCIATION_HOST_PROC_CMD_FULL_PATH   = 10103;
    TELEMETRY_ASSOCIATION_SERVICE_DLL               = 10104;
    TELEMETRY_ASSOCIATION_ARCHIVE_FILE              = 10105;
    TELEMETRY_ASSOCIATION_BROWSER_PROCESS           = 10106;
```

**Review by trend-dennis-tseng — APPROVED** [review]
LGTM

**Pull request merged**